libs/domain/content/link/link.component.ts

```diff
@@ -23,19 +23,34 @@ export class ContentLinkComponent extends ContentMixin<
 
   protected $link = computed(() => {
     const { url, type, id, params } = this.$options();
-    if (url) return url;
+    if (url) return of(url);
     if (type) return this.semanticLinkService.get({ type: type, id, params });
-    return null;
+    return of(undefined);
```

Comment:
Probably better to keep `null`, due to `undefined` can be considered as not initialised 
```suggestion
    return of(null);
```